tmt/steps/execute/__init__.py

```diff
@@ -89,43 +119,54 @@ class ScriptCreatingFile(Script):
     )
 
 TMT_REBOOT_CORE_SCRIPT = Script(
-    path=Path("/usr/local/bin/tmt-reboot-core"),
+    path=effective_scripts_dest_dir() / 'tmt-reboot-core',
     aliases=[],
     related_variables=[])
 
 # Script handling result reporting, in restraint compatible fashion
 TMT_REPORT_RESULT_SCRIPT = ScriptCreatingFile(
-    path=Path("/usr/local/bin/tmt-report-result"),
+    path=effective_scripts_dest_dir() / 'tmt-report-result',
     aliases=[
-        Path("/usr/local/bin/rstrnt-report-result"),
-        Path("/usr/local/bin/rhts-report-result")],
+        effective_scripts_dest_dir() / 'rstrnt-report-result',
+        effective_scripts_dest_dir() / 'rhts-report-result'],
     related_variables=[],
     created_file="tmt-report-results.yaml"
     )
 
 # Script for archiving a file, usable for BEAKERLIB_COMMAND_SUBMIT_LOG
 TMT_FILE_SUBMIT_SCRIPT = Script(
-    path=Path("/usr/local/bin/tmt-file-submit"),
+    path=effective_scripts_dest_dir() / 'tmt-file-submit',
     aliases=[
-        Path("/usr/local/bin/rstrnt-report-log"),
-        Path("/usr/local/bin/rhts-submit-log"),
-        Path("/usr/local/bin/rhts_submit_log")],
+        effective_scripts_dest_dir() / 'rstrnt-report-log',
+        effective_scripts_dest_dir() / 'rhts-submit-log',
+        effective_scripts_dest_dir() / 'rhts_submit_log'],
     related_variables=[]
     )
 
 # Script handling text execution abortion, in restraint compatible fashion
 TMT_ABORT_SCRIPT = ScriptCreatingFile(
-    path=Path("/usr/local/bin/tmt-abort"),
+    path=effective_scripts_dest_dir() / 'tmt-abort',
     aliases=[
-        Path("/usr/local/bin/rstrnt-abort"),
-        Path("/usr/local/bin/rhts-abort")],
+        effective_scripts_dest_dir() / 'rstrnt-abort',
+        effective_scripts_dest_dir() / 'rhts-abort'],
     related_variables=[],
     created_file="abort"
     )
 
+# Profile script for adding SCRIPTS_DEST_DIR to executable pats system-wide
+TMT_ETC_PROFILE_D = ScriptTemplate(
+    path=Path("/etc/profile.d/tmt.sh"),
+    aliases=[],
+    related_variables=[],
+    context={
+        'scripts_dest_dir': str(effective_scripts_dest_dir())
```

Comment:
I'd recommend using upper case for variable names, which seems to be common in the Jinja world in general, and definitely true in our own templates.

Reply:
no objections

Reply:
done, it is now upper case.